web/js/containers/sidebar/layer-list.js

```diff
@@ -86,129 +99,148 @@ class LayerList extends React.Component {
-          </Droppable>
-        </DragDropContext>
-      </div>
+      <LayerRow
+        layer={layer}
+        compareState={compareState}
+        isInProjection={!!projections[projId]}
+        key={id}
+        index={index}
+        zot={zots[id]}
+        names={getNames(id)}
+        isDisabled={!available(id)}
+        isVisible={visible}
+        runningObject={runningDataObj[id]}
+      />
     );
-  }
+  };
+
+  const renderDropdownMenu = () => (
+    <Dropdown className="layer-group-more-options" isOpen={showDropdownMenu} toggle={toggleDropdownMenuVisible}>
+      <DropdownToggle>
+        <FontAwesomeIcon
+          className="layer-group-more"
+          icon="ellipsis-v"
+        />
+      </DropdownToggle>
+      <DropdownMenu>
+        <DropdownItem id="show-all" onClick={() => toggleVisibility(groupLayerIds, true)}>
+          Show All Layers
+        </DropdownItem>
+        <DropdownItem id="hide-all" onClick={() => toggleVisibility(groupLayerIds, false)}>
+          Hide All Layers
+        </DropdownItem>
+        <DropdownItem id="remove-group" onClick={() => removeLayers(groupLayerIds)}>
+          Remove Group
+        </DropdownItem>
+      </DropdownMenu>
+    </Dropdown>
+  );
+
+  return (
+    <div
+      id={`${compareState}-${groupId}`}
+      className="layer-group-case"
+    >
+      <div
+        className="layer-group-header"
+        onMouseEnter={mouseEnter}
+        onMouseLeave={mouseLeave}
+      >
+        <h3 className="layer-group-title">
+          {title}
+          {collapsed ? ` (${layers.length})` : ''}
+        </h3>
+        <div className="layer-group-icons">
+          {showDropdownBtn || isMobile ? renderDropdownMenu() : null}
+          <FontAwesomeIcon
+            className="layer-group-collapse"
+            icon={!collapsed ? 'caret-down' : 'caret-left'}
+            onClick={() => toggleCollapse(groupId, !collapsed)}
+          />
+        </div>
+      </div>
+      <DragDropContext onDragEnd={onDragEnd}>
+        <Droppable
+          droppableId={`${compareState}-${groupId}`}
+          type={`layerGroup${groupId}`}
+          direction="vertical"
+        >
+          {(provided, snapshot) => (
+            <ul
+              className={collapsed ? 'category hidden' : 'category'}
+              ref={provided.innerRef}
+                // eslint-disable-next-line react/jsx-props-no-spreading
+              {...provided.droppableProps}
+            >
+              {layers.map(renderLayer)}
+              {provided.placeholder}
+            </ul>
+          )}
+        </Droppable>
+      </DragDropContext>
+    </div>
+  );
 }
+
 LayerList.propTypes = {
   activeLayers: PropTypes.array,
   available: PropTypes.func,
-  checkerBoardPattern: PropTypes.object,
+  collapsed: PropTypes.bool,
+  compareState: PropTypes.string,
   getNames: PropTypes.func,
   groupId: PropTypes.string,
-  layerGroupName: PropTypes.string,
+  isMobile: PropTypes.bool,
+  launchOpacityModal: PropTypes.func,
```

Comment:
These don't appear to be being used: 
`launchOpacityModal`
`runningLayers`

I feel we had an eslint rule at some point for this, but I've run into accidentally pushing up unused propTypes recently.

Reply:
Hmm yeah I thought we did too but maybe I'm confusing it with what we do have which is the inverse... it yells at you if you have props without corresponding PropTypes 🤔   I think it's worth adding an eslint rule if there is one.